.github/workflows/ci_release.yml

```diff
@@ -0,0 +1,45 @@
+name: CI and Release
+on:
+  push:
+    branches:
+      - main
+    # Trigger on version tags
+    tags:
+      - 'v[0-9]+\.[0-9]+\.[0-9]+'
+      - 'v[0-9]+\.[0-9]+\.[0-9]+-rc(?:[0-9]+|\.[0-9]+)'
```

Comment:
I think we can just use `v*` for simplicity.

Reply:
It was like that at some point :stuck_out_tongue_winking_eye:
Will fix.